### ISSUE TITLE
Angular Gridster instead of jquery-sortable

### ISSUE THREAD

**Issue by sclausen** [DataTorrent/malhar-angular-dashboard#87]
For a more consistent layout I propose [angular-gridster](https://github.com/ManifestWebDesign/angular-gridster) as a replacement for angular-ui-sortable

**Comment by zhx828** [comment]
Agree with above. Would suggest add layout control by users. Packery also a good choice. Glad to help if needed to push some changes.

**Comment by andyperlitch** [comment]
I'm going to give [ng-sortable](https://github.com/a5hik/ng-sortable) a try for v1.0.0... it looks like it has matured a fair amount in the last six months.

**Comment by sclausen** [comment]
ng-sortable hase the same problem as angular-ui-sortable. if the cells are not equal sizes the layout is messed up. for example angular-gridster has relative cell-sizes which makes the layout more consistent.

**Comment by andyperlitch** [comment]
Yeah, I gave it a shot and it was not too pretty... will give gridster another look. My beef with gridster is that it has fixed height for all elements :-/ this can be pretty hard to deal with for certain widget content.

**Comment by andyperlitch** [comment]
removing this from the 1.0.0 milestone as this is probably achievable using a custom dashboard template. If someone else wants to add this option, i am open to PRs.

**Comment by artoliukkonen** [comment]
We choose to switch to Gridster in our project, so I forked this repo and made the changes. It's working pretty fine I'd say. https://github.com/SC5/malhar-angular-dashboard

Disclaimer: haven't touched to tests (possibly broken), haven't touched to demo (completely broken), switched to Foundation instead of Bootstrap (just because we're using it in our project). In case someone chooses to try our version, feel free to contact us for guidance.

**Comment by rihabesx** [comment]
Hi artoliukkonen!
I am interested in the gridster fork but my project uses bootstrap. Could you please tell me what changes are required to make it work as gridster/bootstrap instead of gridster/foundation.

Also what is the possibility of getting the demo application to work. This is simplified version and will help a lot. Thanks.
